project/src/main/puzzle/food-items.gd

```diff
@@ -102,14 +98,7 @@ func add_food_item(cell: Vector2, food_type: int) -> void:
 Callback function which returns the coordinate of the customer's mouth relative to the FoodItems viewport.
 """
 func get_target_pos(customer: Creature) -> Vector2:
-	var target_pos: Vector2
-	# calculate the position within the restaurant scene
-	target_pos = customer.get_node("ChatIconHook").get_global_transform_with_canvas() \
-			.xform(MOUTH_POSITIONS_BY_ORIENTATION[customer.get_orientation()] * customer.creature_visuals.scale.y)
-	# calculate the position within the customer viewport
-	target_pos = _customer_view_viewport.canvas_transform.xform(target_pos)
-	# calculate the position within the customer viewport texture
-	target_pos = _customer_view.get_global_transform_with_canvas().xform(target_pos / _customer_view.stretch_shrink)
+	var target_pos: Vector2 = _restaurant_view.get_customer_mouth_position(customer)
```

Comment:
This change is likely to make `_customer_view` and `_customer_view_viewport` unused fields

Reply:
Removed `_customer_view`, `_customer_view_viewport`